### ISSUE TITLE
Packets deny still getting through

### ISSUE THREAD

**Issue by x1p** [kangarko/ChatControl-Pro#945]
I have a several rules in my packets who are getting caught which should be denied. This way it won't reach the player. Well, thats the plan.

The rule is getting caught. But still let through.
What am I missing?

Normally I would have:
```
match ^The line I want to dismiss
dont verbose
then deny
```
But somehow, messages still got through. Because I wasn't sure about my line in case a made a mistake, I disabled `dont verbose`. To see the message on the console.
The message is getting caught. But still gets through.
Using
ChatControl 7.9.27
Protocollib build 421
Plotsquared 18.08.12-4b8c434-2035
I have this on different versions of ChatControl and Protocollib. So I suspect is has something to do with the priority plotsquared uses. ChatControl priorities:
```
Listener_Priority:
  Checker: LOWEST
  Formatter: NORMAL
  Signs: HIGHEST
```

**Comment by kangarko** [comment]
A developer is looking into it, may take up to 48 hours.

**Comment by jascotty2** [comment]
I have attempted several different ways to try to reproduce the issue you're seeing, but everything I've tried works as expected.
Setting `Checker: NORMAL` allows PlotSquared to take precedence as described, but setting it to LOWEST forces it to run before PlotSquared has a chance to see it.

Are you certain that no other plugin on the server is messing with chat events?

**Comment by x1p** [comment]
It is the message that you can't build on another ones plot or on the road or the missing permissions message. When you click fast enough to spam a lot of messages, it will eventual come through.

I solved it for now by deleting the entire message. The downside of this, is that staff does not get the warning what the exact problem is.

**Comment by x1p** [comment]
The message: `Have an admin set the flag`

The rule:
```
match Have an admin set the flag
ignore perm plots.admin.interact.other
then deny
```

This message will be shown by PlotSquared when a player tries to break/build on a plot which is not his own.

Same goes for the line when a player tries to break/build on the road.
I will do some further testing myself too.

**Comment by jascotty2** [comment]
I was able to reproduce with that additional information - thanks!

I found the source of the issue, and this will be fixed in the next release (7.9.31)
